rollup/circuitcapacitychecker/libzkp/Cargo.toml

```diff
@@ -20,7 +20,7 @@ maingate = { git = "https://github.com/scroll-tech/halo2wrong", branch = "halo2-
 halo2curves = { git = "https://github.com/scroll-tech/halo2curves.git", branch = "0.3.1-derive-serde" }
 
 [dependencies]
-prover = { git = "https://github.com/scroll-tech/zkevm-circuits.git", tag = "v0.9.10", default-features = false, features = ["parallel_syn", "scroll", "shanghai", "strict-ccc"] }
+prover = { git = "https://github.com/scroll-tech/zkevm-circuits.git", tag = "v0.9.13", default-features = false, features = ["parallel_syn", "scroll", "shanghai", "strict-ccc"] }
```

Comment:
I want to confirm the following:
- This CCC version will not reject SHA-256,
- This CCC version will be compatible with these 3 networks: Scroll Sepolia (pre-Bernoulli), Scroll Sepolia (post-Bernoulli), and Scroll mainnet.